dev-docs/source/GettingStarted.rst

```diff
@@ -92,22 +92,30 @@ if you wish to setup eclipse for use developing mantid, then instructions can be
 
 Ubuntu 20.04
 ~~~~~~~~~~~~
+- Mantid uses `qtpy` to talk to Python bindings of Qt.  It is recommended to have the _
```

Comment:
Is this necessary? I don't have this set and it's okay for me.

Reply:
It depends on how `qtpy` is configured on the system.  On my testing env, I can start mantidworkbench without any issue.  However, the following unit test will fail with a "No Qt bindings could be found" error if QT_API is not specified system wide during unit testing, for example

* python.plots.plots__init__Test.plots__init__Test
* python.plots.surfacecontourplotsTest.surfacecontourplotsTest
* python.scripts.StitchingTest.StitchingTest 
* PythonTOFTOFReduction.TOFTOFGUITest.TOFTOFGUITest

Reply:
@martyngigg I had the same issue as @KedoKudo. I would keep this in until we get rid of PyQt4 and we fix that in the launch script

Reply:
Thanks both. I must have not run those particular unit tests. I'm happy to keep it until we remove Qt4.

We just need to bump the version in `ns-control` for the developer package so that it looks like an update to the package manager when it comes along.